mvg/mvg.py

```diff
@@ -765,6 +765,111 @@ def get_analysis_results(self, request_id: str) -> dict:
 
         return response.json()
 
+    # Labels
+    def create_label(
+        self,
+        sid: str,
+        timestamp: int,
+        label: str,
```

Comment:
This is not according to the API specification, which lacks severity and names the notes "meta" which also is a dictionary.
I think the severity should be part of "meta" as for labels not related to failures severity does not make any sense.

Reply:
When there is no issue with the label, severity level is zero. Severity level is always mandatory because we want to be able to create health index by taking expected value of the severity. It is mandatory there not part of meta data.